tests/contrib/test_rest_framework_dataclasses.py

```diff
@@ -0,0 +1,50 @@
+import pytest
+import typing
+
+from dataclasses import dataclass
+from django.urls import path
+from drf_spectacular.utils import extend_schema
+from rest_framework.decorators import api_view
+from rest_framework_dataclasses.serializers import DataclassSerializer
+
+from tests import assert_schema, generate_schema
+
+
+@dataclass
+class Person:
+    name: str
+    length: int
+
+
+@dataclass
+class Group:
+    name: str
+    leader: Person
+    members: typing.List[Person]
+
+
+class GroupSerializer(DataclassSerializer):
+    class Meta:
+        dataclass = Group
+
+
```

Comment:
to make the suite run through you probably need this also. the tests may skip but not break.

`@pytest.mark.skipif(sys.version_info < (3, 7), reason='dataclass required by package')`

Reply:
Should all be fixed now.